packages/vuetify/src/components/VTextarea/__tests__/VTextarea.spec.cy.tsx

```diff
@@ -27,8 +27,8 @@ describe('VTextarea', () => {
       .should(el => expect(el.outerHeight()).to.equal(56))
       .click()
       .type('Lorem ipsum dolor sit amet consectetur adipisicing elit. ')
-      .should(el => expect(el.outerHeight()).to.equal(80))
+      .should(el => expect(el.outerHeight()).to.equal(78))
```

Comment:
When it comes two lines, it should be

height(78px) = line-height (24px) * 2 + padding top/bottom (30px)